README.md

```diff
@@ -439,10 +439,9 @@ AWSXRayRecorder.Instance.AddAnnotation("mykey", "my value");
 AWSXRayRecorder.Instance.AddMetadata("my key", "my value");
 ```
 
-### AWS Lambda support (.NET Core)
-
-You can create `Subsegment` inside lambda function.     
-*Note*: The AWS Lambda execution environment creates a `Segment` before the Lambda function is invoked, so a `Segment` cannot be created inside the Lambda function.
+### AWS Lambda support (.NET Core) 
+You can only create and close `Subsegment` inside a lambda function.     
+*Note*: The AWS Lambda execution environment creates a `FacadeSegment` before the Lambda function is invoked, so a `Segment` cannot be created inside the Lambda function.
```

Comment:
This can be -
1. The AWS Lambda execution environment creates a Segment before each Lambda function invocation and sends it to X-Ray service. AWS X-Ray .NET/Core SDK will make sure there will be a FacadeSegment inside lambda context so that customers are able to instrument their application successfully for subsegments. Subsegments generated inside Lambda function are attached to this `FacadeSegment`  and only subsegments are streamed by the SDK.
2. `Segment` cannot be created inside the Lambda function. All operations on `Segment` will throw an exception (check the code and mention the exception)